checker.cpp

```diff
@@ -1,17 +1,100 @@
 #include <assert.h>
+#include <iostream>
 
-bool vitalsAreOk(float bpm, float spo2, float respRate) {
-  if(bpm < 70 || bpm > 150) {
-    return false;
-  } else if(spo2 < 90) {
-    return false;
-  } else if(respRate < 30 || respRate > 95) {
-    return false;
-  }
-  return true;
+struct Vitals {
+	//int bedNumber ;
```

Comment:
Very impressive thinking! A bed number is an important part of the story, which needs to be included in the alert. But why is it commented out?